typescript/src/languages/en-US/commands/starwars.json

```diff
@@ -0,0 +1,200 @@
+{
+    "filmDescription": "Get data for any given Star Wars movie",
+    "filmExtended": {
+        "usages": [
+            "Movie"
+        ],
+        "extendedHelp": "Uses a fuzzy search to match on both movie episode ID (1, 2...6) as well as movie title.",
+        "explainedUsage": [
+			[
+                "Movie",
+                "The movie for which you want to find data"
+			]
+		],
+        "examples": [
+			"1",
+			"hope",
+            "revenge of the sith"
+		],
+        "reminder": "At the moment only data on the first 6 movies is available. Data in movies 7 through 9 and spin-offs might be added in the future."
+    },
+    "filmQueryFail": "{{REDCROSS}} I am sorry, but that query failed. Are you sure `{{film}}` is actually a Star Wars movie?",
```

Comment:
might be a good shout to remind that only 1-6 movies are available via the API, at the moment.